specs/trust-store-trust-policy.md

```diff
@@ -368,17 +368,17 @@ There are two types of CRLs (per RFC 3280), Base CRLs and Delta CRls.
   Delta CRLs are signed by the certificate issuing CAs.
 - **Indirect CRLs:** Special Case in which BaseCRLs and Delta CRLs are not signed by the certificate issuing CAs, instead they are signed with a completely different certificate.
 
-Notary v2 MUST support BaseCRLs and Delta CRLs.
-Notary v2 MAY support Indirect CRLs.
-Notary v2 supports only HTTP CRL URLs.
+Notary Project specification implementation MUST support BaseCRLs and Delta CRLs.
```

Comment:
The before each Notary Project

Reply:
update it to `Implementations of the Notary Project specification`